web/react/utils/utils.jsx

```diff
@@ -1241,3 +1241,25 @@ export function getPostTerm(post) {
 
     return postTerm;
 }
+
+// taken from https://davidwalsh.name/javascript-debounce-function
+// should it be replaced by lodash.debounce ?
+export function debounce(func, wait, immediate) {
```

Comment:
Any external code or 3rd party lib should be called out so we can review licensing.